resharper/src/resharper-unity/CSharp/Daemon/Stages/Analysis/TypeInsteadOfStringAnalyzer.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System.Collections.Generic;
+using JetBrains.Annotations;
+using JetBrains.ReSharper.Daemon.BuildScripts.DaemonStage.Highlightings;
+using JetBrains.ReSharper.Feature.Services.Daemon;
+using JetBrains.ReSharper.Plugins.Unity.CSharp.Daemon.Errors;
+using JetBrains.ReSharper.Plugins.Unity.CSharp.Daemon.Stages.Dispatcher;
+using JetBrains.ReSharper.Psi;
+using JetBrains.ReSharper.Psi.CSharp;
+using JetBrains.ReSharper.Psi.CSharp.Impl.DocComments;
+using JetBrains.ReSharper.Psi.CSharp.Parsing;
+using JetBrains.ReSharper.Psi.CSharp.Tree;
+using JetBrains.ReSharper.Psi.CSharp.Util.Literals;
+using JetBrains.ReSharper.Psi.Resolve;
+using JetBrains.ReSharper.Psi.Tree;
+using JetBrains.ReSharper.Psi.Util;
+
+namespace JetBrains.ReSharper.Plugins.Unity.CSharp.Daemon.Stages.Analysis
+{
+    [ElementProblemAnalyzer(typeof(IInvocationExpression), HighlightingTypes =
+        new[] {typeof(TypeInsteadOfStringUsingWarning), typeof(InvalidStringForTypeWarning)})]
+    public class TypeInsteadStringAnalyze : UnityElementProblemAnalyzer<IInvocationExpression>
```

Comment:
`UseExplicitTypeInsteadOfString` would be better IMO. 
@citizenmatt WDYT?

Reply:
I think it's better too, so I renamed file.